plugins/login-resources/src/utils.ts

```diff
@@ -741,13 +731,12 @@ export async function afterConfirm (clearQuery = false): Promise<void> {
   if (joinedWS.length === 0) {
     goTo('createWorkspace', clearQuery)
   } else if (joinedWS.length === 1) {
-    const result = (await selectWorkspace(joinedWS[0].uuid, null))[1]
+    const result = (await selectWorkspace(joinedWS[0].url, null))[1]
```

Comment:
why url?

Reply:
because we use workspace url in `selectWorkspace`, it was a mistake in the global accounts PR

Reply:
ok